Gemfile.lock

```diff
@@ -82,21 +82,21 @@ GEM
     ast (2.4.2)
     autoprefixer-rails (10.2.4.0)
       execjs
-    aws-eventstream (1.1.0)
-    aws-partitions (1.425.0)
-    aws-sdk-core (3.112.0)
+    aws-eventstream (1.1.1)
+    aws-partitions (1.445.0)
+    aws-sdk-core (3.114.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.42.0)
+    aws-sdk-kms (1.43.0)
```

Comment:
### [aws-sdk-kms](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.42.0 → 1.43.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms)_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/v1.42.0...v1.43.0).

These are the first 10 commits:
- (3f069bc) [update sample code to use the default credential provider in…](https://github.com/aws/aws-sdk-ruby/commit/3f069bc5bcdbba5679f7629bac140ab96892c485)
- (8759f0f) [Clarify log_level configuration docs](https://github.com/aws/aws-sdk-ruby/commit/8759f0f22381baffee955548249dd5d98cd25fe2)
- (f2a7c5d) [Add DBSecurityGroupName to AuthorizeDBSecurityGroupIngress](https://github.com/aws/aws-sdk-ruby/commit/f2a7c5df66f4c2ea88550e7fbf769ba094bf5367)
- (531d043) [Modified documentation of `option[:instance_tenancy]`](https://github.com/aws/aws-sdk-ruby/commit/531d0433311cc41a06e36d144cedac0c1f64253f)
- (f7acaca) [Fix comment on WorkflowExecutionCollection#with_status and #…](https://github.com/aws/aws-sdk-ruby/commit/f7acaca9627ec0b6865bd0887cef61a96066bd25)
- (a149796) [Merge pull request #555 from juno/fix-comment-on-swf](https://github.com/aws/aws-sdk-ruby/commit/a149796efc7a1cdc328295e8ddf2650ec95f51b1)
- (a476c09) [Merge pull request #554 from asonas/modified_documentation_o…](https://github.com/aws/aws-sdk-ruby/commit/a476c0995273a6d47ceda8a0dd1a3d84b87a6387)
- (cb6d9e0) [Merge pull request #553 from joelr/patch-1](https://github.com/aws/aws-sdk-ruby/commit/cb6d9e047ba6e60a781c2d56b64628e7b9ed7e6c)
- (5ca4709) [Merge pull request #542 from gburt/master](https://github.com/aws/aws-sdk-ruby/commit/5ca4709d596b3502277984125e911dd83d96a0a6)
- (9ef3428) [Merge pull request #541 from EronHennessey/samples-use-def-p…](https://github.com/aws/aws-sdk-ruby/commit/9ef34287527e53ce144480ef2187e65981d5862e)


</details>